CHANGELOG.md

```diff
@@ -5,11 +5,17 @@ In this file, all the changes are listed by release.
 
 * Summary of v.next changes go here.
 * Added a CHANGELOG.md
+## 100.13
+* Basemapgallery QtWidgets.
+* BasemapgalleryController modified to only load geomodel's supported basemaps.
```

Comment:
might a bit too specific.
could be `BasemapgalleryController matching IOS loading supported basemaps functionalities`